core/src/test/scala/wartremover/warts/Any2StringAddTest.scala

```diff
@@ -6,13 +6,40 @@ import org.scalatest.FunSuite
 import org.wartremover.warts.Any2StringAdd
 
 class Any2StringAddTest extends FunSuite {
-  test("disable any2stringadd") {
+  test("Implicit conversion to string is disabled") {
     val result = WartTestTraverser(Any2StringAdd) {
-      {} + "lol"
+      val one = 1
+      val two = "2"
+      one + two
+      two + one
     }
-    assertResult(List("Scala inserted an any2stringadd call"), "result.errors")(result.errors)
+    assertResult(List.fill(2)("Implicit conversion to string is disabled"), "result.errors")(result.errors)
     assertResult(List.empty, "result.warnings")(result.warnings)
   }
+
+  test("String literals are allowed") {
+    val result = WartTestTraverser(Any2StringAdd) {
+      1 + "a"
+      "a" + 1
```

Comment:
@ChrisNeveu Sorry, I thought that you [answered](https://github.com/puffnfresh/wartremover/pull/276#issuecomment-255789483) on my initial [question](https://github.com/puffnfresh/wartremover/pull/276#issue-184878570) about string literals. I'll update the PR.